### ISSUE TITLE
Monitor Information/Dimensions

### ISSUE THREAD

**Issue by TheXadGaming** [QB64-Phoenix-Edition/QB64pe#360]
A way to get a monitor's size would be useful, I'm not sure about all the fine details on how to specify which monitor, and other problems like that. A Set of things for more info about the monitor would be useful as well.
But it could be a value like this:
`monitornumber = 1 'Which Monitor to get the dimensions of.`
`MonitorWidth = _monitorwidth(monitornumber)`
`MonitorHeight = _monitorheight(monitornumber)`

**Comment by a740g** [comment]
@TheXadGaming I think this was a legit and useful feature request and you should have left it open. Not sure why you closed it.

**Comment by TheXadGaming** [comment]
It is a good feature, but QB64 does already have some of this functionality: `_DESKTOPWIDTH` and `_DESKTOPHEIGHT`. I guess all we'd need is a way to distinguish between monitors connected to the PC.

**Comment by a740g** [comment]
> It is a good feature, but QB64 does already have some of this functionality: `_DESKTOPWIDTH` and `_DESKTOPHEIGHT`. I guess all we'd need is a way to distinguish between monitors connected to the PC. These functions do say the say the information of the "current desktop", so I assume whichever one the program window is in, or whichever monitor is in focus right then.

Yeah. I was also thinking the same thing and would like to see something like the following implemented.

```vb
_SETWINDOWMONITOR monitor&     ' Set monitor for the QB64 window
monitors& = _MONITORCOUNT&     ' Get number of connected monitors
x& = _DESKTOPWIDTH [(monitor&)]  ' Get specified monitor width (current video mode used by monitor)
y& = _DESKTOPHEIGHT [(monitor&)] ' Get specified monitor height (current video mode used by monitor)
```

> 
> My skills at large projects, like QB64PE, and C++, aren't very good, so if someone would like to add this.

No worries. We periodically review these feature requests, and whenever one of us has the time and motivation, we address some of them. Thank you for reopening the issue.
